packages/network-controller/src/NetworkController.test.ts

```diff
@@ -3722,6 +3724,103 @@ describe('NetworkController', () => {
       });
     });
   });
+
+  describe('upsertNetworkConfiguration', () => {
+    it('add a network configuration', async () => {
```

Comment:
When does this method add a network configuration?

Reply:
🤔 [here](https://github.com/MetaMask/core/pull/1064/files#diff-ea0aa37793484e278bd59970b7a6ce8380b75b723b34612081d402fb66261cfdR505)?

Reply:
Right, sorry, I should have asked the question better. Wondering if it would make sense to clarify when it adds a network configuration, as the way this is worded sounds like this is the default or most obvious behavior, when it's simply one of two paths that the code can take. So, something like:

``` suggestion
    it('adds a network configuration when the configuration being added does not have an rpcURL that matches an existing configuration', async () => {
```

Or, reworded slightly:

``` suggestion
    it('adds the given network configuration when its rpcURL does not match an existing configuration', async () => {
```